src/nvim/auevents.lua

```diff
@@ -79,7 +79,7 @@ return {
     'TabNew',                 -- when creating a new tab
     'TabNewEntered',          -- after entering a new tab
     'TermChanged',            -- after changing 'term'
-    'TermClose',              -- after the processs exits
+    'TermClose',              -- when the process exits
```

Comment:
This wording is really unfortunate imho. Often, one really wants to know if your code is executed pre-event or post-event, and "when" doesn't specify that. If it's "after" or "before", it should be specified here and in the docs, and if it can't be specified (because concurrency or whatnot), make a note of it.